db/seeds.rb

```diff
@@ -45,20 +45,20 @@ def generate_items
 
 
   def generate_users
-    users = User.create([
-            { name: "Rachel Warbelow", email_address: "demo+rachel@jumpstartlab.com", password: "password"},
-            { name: "Jeff Casimir", email_address: "demo+jeff@jumpstartlab.com", password: "password", display_name: "j3"},
-            { name: "Jorge Tellez", email_address: "demo+jorge@jumpstartlab.com", password: "password", display_name: "novohispano"},
-            { name: "Josh Cheek", email_address: "demo+josh@jumpstartlab.com", password: "password", display_name: "josh", role: 1}
+    @users = User.create([
+            { full_name: "Rachel Warbelow", email: "demo+rachel@jumpstartlab.com", password: "password"},
```

Comment:
Line is too long. [105/80]<br>Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.<br>Space inside } missing.